src/SignalR/clients/ts/signalr/src/Utils.ts

```diff
@@ -37,18 +37,23 @@ export class Arg {
 export class Platform {
     // react-native has a window but no document so we should check both
     public static get isBrowser(): boolean {
-        return typeof window === "object" && !!window.document;
+        return typeof window === "object" && typeof window.document === "object";
     }
 
     // WebWorkers don't have a window object so the isBrowser check would fail
     public static get isWebWorker(): boolean {
         return typeof self === "object" && "importScripts" in self;
     }
 
+    // react-native has a window but no document
+    static get isReactNative(): boolean {
+        return typeof window === "object" && window.document === undefined;
```

Comment:
```suggestion
        return typeof window === "object" && typeof window.document === "undefined";
```

Why not this for consistency with all the other checks?